src/main/java9/module-info.java

```diff
@@ -36,20 +36,24 @@
   exports org.assertj.core.util.introspection;
   exports org.assertj.core.util.xml;
 
-  requires java.instrument;
+
+  requires static java.sql; // import java.sql.Timestamp; in Assertions.java for Javadoc-only
```

Comment:
https://github.com/joel-costigliola/assertj-core/blob/master/src/main/java/org/assertj/core/api/Assertions.java#L2552 can be replaced by
```
   * <li><code>yyyy-MM-dd HH:mm:ss.SSS</code> (for {@code java.sql.Timestamp} String representation support)</li>
```

Reply:
True.

Reply:
Not needed anymore since 5c136d6268a6e0c375c6d6ae266c6a6e2b2b0c6d.